### PR TITLE
INFRA-892 : Support remote connection to PG server

### PR DESCRIPTION
**Pull request by dud225**

### CODE REVIEW

**Review by thomsh — APPROVED** [review]
LGTM
One drawback is to install the postgresql client package on the server to perform remote action as it's not a pure ruby implem

**Comment by dud225** [comment]
Yes that's the way the community cookbook is implemented. Using a Ruby gem may involve compiling so that may not be a better idea.

**Comment by rottenbytes** [comment]
I confirm it's worst. been there done that

**Comment by thomsh** [comment]
> Yes that's the way the community cookbook is implemented. Using a Ruby gem may involve compiling so that may not be a better idea.

Yes, it's a job for another cookbook as we want to keep using this one I'm okay :)

File: resources/user.rb [hunk 1]
```diff
@@ -28,29 +28,33 @@
 property :attributes,         Hash, default: {}
 
 # Connection prefernces
```
Comment (zborg) [inline resources/user.rb]: typo

**Review by zborg — APPROVED** [review]
LGTM

File: resources/user.rb (hunk 1 above)
Comment (dud225) [reply resources/user.rb]: thanks

**Pull request merged**